cli/src/main/java/org/jboss/as/cli/handlers/OperationRequestHandler.java

```diff
@@ -87,6 +88,7 @@ public void handle(CommandContext ctx) throws CommandLineException {
         }
 
         try {
+            CliLogger.ROOT_LOGGER.executingOperationRequest(op.getOperation().toString());
```

Comment:
This is okay, but debug logging isn't required in a logger interface. Also {{toString()}} shouldn't be required. If debug is enabled the logger will essentially us `String.valueOf()` for the `op.getOperation()` value.